AISKU/Tests/applicationinsights.e2e.tests.ts

```diff
@@ -280,6 +306,38 @@ export class ApplicationInsightsTests extends TestClass {
             }].concat(this.asserts(1))
         });
 
+        this.testCaseAsync({
+            name: 'E2E.GenericTests: trackException with message holding error sends to backend with custom properties',
+            stepDelay: 1,
+            steps: [() => {
+                let exception: Error = null;
+                try {
+                    window['a']['b']();
+                    Assert.ok(false, 'trackException test not run');
+                } catch (e) {
+                    // Simulating window.onerror option
+                    let autoTelemetry = {
+                        message: e,
+                        url: "https://dummy.error.example.com",
+                        lineNumber: 42,
+                        columnNumber: 53,
+                        error: undefined,
+                        evt: null
+                    } as IAutoExceptionTelemetry;
+    
+                    try {
+                        exception = e;
+                        this._ai.trackException({ exception: autoTelemetry }, { custom: "custom value" });
+                    } catch (e) {
+                        console.log(e);
```

Comment:
nit: Are the console logs on purpose? But I guess it's fine to have in tests.

Reply:
Assert.ok(false) fails the test correct? and the stack is displayed as the error message.  What is the purpose of console.log?

Reply:
Correct, console.log is for easier debugging the failure in a browser.